api/gateway.proto

```diff
@@ -285,3 +290,18 @@ message GatewayConnectionStats {
   // Statistics for each sub band.
   repeated SubBand sub_bands = 10;
 }
+
+// The plain text value of the gateway secret.
+message GatewaySecretPlainText{
+  map<string,bytes> values = 1 [(validate.rules).map.keys.string = {pattern: "^[a-z0-9](?:[-]?[a-z0-9]){2,}$" , max_len: 36}];
+}
```

Comment:
Shouldn't we have a map with keyed secrets per entity, whose value is a marshaled message?

Reply:
So then, `Gateway` can contain multiple independent secrets, keyed by string, and one of them is `basicstation`, which is the encrypted marshaled `GatewayBasicStationSecretData` which has two string fields